### PR TITLE
{lib}[GCCcore/9.3.0] tbb v2020.1

### PR DESCRIPTION
**Pull request by bioteamMichael**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/tbb/tbb-2020.1-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+source_urls = ['https://github.com/oneapi-src/oneTBB/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7c96a150ed22bc3c6628bc3fef9ed475c00887b26d37bca61518d76a56510971']
+
+builddependencies = [
+    ('binutils', '2.34')
+]
+
+moduleclass = 'lib'
+
```
Comment (lexming) [inline easybuild/easyconfigs/t/tbb/tbb-2020.1-GCCcore-9.3.0.eb]: This blank line at the end of the file should be removed for the checks to pass
```suggestion
```

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/208129ce048a4eb2b5661cfe6bf0af49 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @bioteamMichael !

**Comment by ocaisa** [comment]
We now have the same tbb in 2020a twice, once as `oneTBB` (see #10121) and again as `tbb`

**Pull request merged**